wandb/integration/openai/__init__.py

```diff
@@ -1,3 +1,12 @@
-__all__ = ("autolog",)
+__all__ = ("autolog", "WandbLogger")
+
+from wandb import util
+
+_openai = util.get_module("openai")
+
+from pkg_resources import parse_version
 
 from .openai import autolog
+
+if parse_version(_openai.__version__) > parse_version("0.28.1"):
```

Comment:
what happens is openai is not installed this line wold fail, since `_openai` will be none!

Reply:
wrapping in with if-else would fix this, imo.